napi-inl.h

```diff
@@ -1791,9 +1834,8 @@ inline ClassPropertyDescriptor<T> ObjectWrap<T>::InstanceMethod(
     InstanceMethodCallback method,
     napi_property_attributes attributes,
     void* data) {
-  CallbackData* callbackData = new CallbackData({}); // TODO: Delete when the class is destroyed
-  callbackData->instanceMethodCallback = method;
-  callbackData->data = data;
+  // TODO: Delete when the class is destroyed
+  InstanceMethodCallbackData* callbackData = new InstanceMethodCallbackData({ method, data });
```

Comment:
_Why_ `auto`? I don't usually prefer it, personally. Is there a Node.js style guide that suggests when to use it?